src/components/event/EventHeader.astro

```diff
@@ -26,17 +45,71 @@ const end = frontmatter.end;
+                                                    {location.locationLatLng && (
+                                                        <a
+                                                            class="m-1 btn btn-sm btn-outline-light"
+                                                            href={`https://www.openstreetmap.org/?mlat=${location.locationLatLng[0]}&mlon=${location.locationLatLng[1]}&zoom=16#map=16/${location.locationLatLng[0]}/${location.locationLatLng[1]}`}
+                                                            target="_blank"
+                                                        >
+                                                            <i class="fa-duotone fa-map-pin me-1" />
+                                                            See on map
+                                                        </a>
+                                                    )}
+                                                    :
+                                                </strong>
+                                                <ul class="list-unstyled">
+                                                    <li>
+                                                        {locationURLs && (
+                                                            <ul>
+                                                                {locationURLs.map((loc) => {
+                                                                    return (
+                                                                        <li class="list-item">
+                                                                            <a class="text-white underline" href={loc}>
+                                                                                {loc}
+                                                                            </a>
+                                                                        </li>
+                                                                    );
+                                                                })}
+                                                            </ul>
+                                                        )}
+                                                    </li>
+                                                </ul>
+                                            </li>
+                                        </ul>
+                                    </li>
+                                );
+                            })
+                        }
+                    </ul>
+                </dd>
+
                 {
-                    frontmatter.locationURL && (
+                    frontmatter.links && frontmatter.links.length > 0 && (
                         <>
-                            <dt>{frontmatter.locationURL.length === 1 ? 'Link' : 'Links'}:</dt>
+                            <dt>Links:</dt>
```

Comment:
could we have "Link" if `frontmatter.links.length==1`

Reply:
Yeap! I got rid of it after tinkering around: https://github.com/nf-core/website/pull/2376/commits/7c7120d341a289eec67e1eb12e277b59f9ff10cf